src/lib/DataTableHeader/DataTableRow.tsx

```diff
@@ -22,14 +23,62 @@ export function DataTableRow<T, TRouteNames>({
   columns,
   actions,
   rowStyle,
+  rowHighlight,
   actionsPosition,
 }: DataTableRowProps<T, TRouteNames>) {
   const keyValue = getDeepValue(record, keyField);
   const [collapsed, setCollapsed] = useState(true);
 
+  const operator_table = {
+    ">": function (a: number | Date, b: number | Date) {
+      return a > b;
+    },
+    "<": function (a: number | Date, b: number | Date) {
+      return a < b;
+    },
+    "==": function (a: number | Date, b: number | Date) {
+      return a == b;
+    },
+    "!=": function (a: number | Date, b: number | Date) {
+      return a != b;
+    },
+  };
+
+  function getStyle(rowObjectT: T, rowHighlight?: RowHighlightInterface<T>): CSSProperties | undefined {
+    const defaultStyle: CSSProperties = {
+      backgroundColor: "rgba(255,0,0,0.7)",
+      color: "white",
+    };
+
+    if (!rowHighlight) {
+      return undefined;
+    }
+
+    let selectedValue: number | Date | undefined;
+    if (typeof rowObjectT[rowHighlight.compareField] == "number") {
+      selectedValue = rowObjectT[rowHighlight.compareField] as unknown as number;
+    } else if (typeof rowObjectT[rowHighlight.compareField] == "string") {
+      selectedValue = new Date(rowObjectT[rowHighlight.compareField] as unknown as string);
+    }
+
+    if (selectedValue == null) {
+      return undefined;
+    }
+
+    if (typeof selectedValue != "number" && isNaN(selectedValue?.getDate())) {
+      return undefined;
+    }
+
+    if (operator_table[rowHighlight.operation](selectedValue, rowHighlight.compareValue)) {
+      return rowHighlight.customStyle ? rowHighlight.customStyle : defaultStyle;
```

Comment:
its not wrong at all, just reads a little difficult, for the future i think also rowHighlight.customStyle ?? defaultStyle would be possible (personal opinion :D)

Reply:
I changed it with ?? . I know the condition is a little hard to read, but it's the only way to make it customizable.

Reply:
oh awesome many thanks, as i said,  it was just an opinion of me, not that you have to and you also dont have to change if you dont agree.
complex logic is often hard to read, most important is that it works 😄